content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -34,7 +34,7 @@ <h3>Kubernetes Deployments</h3>
                 master schedules mentioned application instances onto individual Nodes in the cluster.
                 </p>
 
-                <p>Once the application instances are created, a Kubernetes Deployment Controller continuously monitors those instances. If the Node hosting an instance goes down or is deleted, the Deployment controller replaces it. <b>This provides a self-healing mechanism to address machine failure or maintenance.</b></p>
+                <p>Once the application instances are created, a Kubernetes Deployment Controller continuously monitors those instances. If the Node hosting an instance goes down or is deleted, the Deployment controller replaces the instance with an instance on another Node in the cluster. <b>This provides a self-healing mechanism to address machine failure or maintenance.</b></p>
```

Comment:
```suggestion
                <p>Once the application instances (pods) are created, a Kubernetes Deployment Controller continuously monitors those pods. If the Node hosting an pod goes down or is deleted, the Deployment controller replaces the failed pod, which the scheduler will place onto another, working Node in the cluster. <b>This provides a self-healing mechanism to address machine failure or maintenance.</b></p>
```

Reply:
@sftim Thank you for the suggestion. 

Like mentioned by @zparnold before, since this is very basic and intro level doc (Also to maintain consistency), it's better to leave it as `instance`.

Reply:
Will incorporate ```on other working node in the cluster``` from the suggestion.

Reply:
The concern I have is for people who think of “instances” as the servers they deploy applications onto. If such a server fails, the Deployment controller does _not_ replace it.

For example, the world's most popular utility computing provider, Amazon Web Services<sup>[ref](https://www.canalys.com/newsroom/cloud-market-share-q4-2018-and-full-year-2018)</sup>, call their compute servers “instances”.